src/openzaak/components/catalogi/models/informatieobjecttype.py

```diff
@@ -62,3 +67,12 @@ def __str__(self):
         if self.concept:
             representation = "{} (CONCEPT)".format(representation)
         return representation
+
+    def save(self, *args, **kwargs):
+        if not self.pk:
+            transaction.on_commit(AutorisatieSpec.sync)
```

Comment:
This only makes sense if the save is wrapped in a transaction.atomic right?

Reply:
Or an outer block of code is wrapped in atomic transaction, which the admin change views are

Reply:
Code updated!